### DISCUSSION TITLE
Students

### DISCUSSION THREAD

**Discussion by Nivi-A** [github/india#295] [General]
DOUBT IN WEBSITE 
I clicked submit option for GitHub , LinkedIn and AICTE links but it didn't show up "submitted successfully" message
it worked fine while submitting Microsoft Learn and Future skill prime but not for the other 3
I tried in my friend's laptop , it did and I had access to all vids 
and then I tried logging in from my lap , I couldn't access 
Please help me out

**Comment by Kamaruddheen** [comment]
Are you using Google Chrome browser (i hope not) if so then use **Microsoft Edge/Firefox** because in Google Chrome some features are disabled i think...

**Reply by Nivi-A** [reply]
yeah , I was using chrome , then I tried in incognito tab it worked fine 
thank you so much , I will use Microsoft edge/Firefox hereafter .

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
